pandas/tests/interchange/test_impl.py

```diff
@@ -295,3 +295,19 @@ def test_datetimetzdtype(tz, unit):
     )
     df = pd.DataFrame({"ts_tz": tz_data})
     tm.assert_frame_equal(df, from_dataframe(df.__dataframe__()))
+
+
+def test_interchange_from_non_pandas_tz_aware():
+    # GH 54239
+    import pyarrow as pa
```

Comment:
```suggestion
    pa = pytest.importorskip("pyarrow")
```

Reply:
thanks, I tried it, but still have the same error.

Reply:
This will help CI builds that don't have pyarrow installed

Reply:
ahh, that is the reason why many checks failed